### DISCUSSION TITLE
ChatGPT Model

### DISCUSSION THREAD

**Discussion by PurplePrawns** [karakeep-app/karakeep#348] [General]
OpenAI is suggesting to use gpt-4o-mini to replace gpt-3.5-turbo. I've switched that in the env but anyone have qualitative measurements? OpenAI is saying it's quicker and cheaper -- and 3.5 Turbo use should be deprecated.

**Comment by MohamedBassem** [comment]
Hi, this is answered in https://github.com/hoarder-app/hoarder/issues/334. TL;DR We'll use 4o-mini in the next releae.
